apps/demos/Demos/Scheduler/Templates/description.md

```diff
@@ -1,10 +1,14 @@
-The following DevExtreme Scheduler properties can be used to specify custom templates (both globally and for individual views):
+You can double-click a DevExtreme Scheduler appointment to open an edit form from the UI. Scheduler allows you to customize the form: rearrange items, create custom fields, and specify popup settings.
 
-* Appointment rectangle: [appointmentTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#appointmentTemplate) / **views[]**.[appointmentTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#appointmentTemplate)
+The form includes two groups: `mainGroup` (general information) and `recurrenceGroup` (recurrence settings). Scheduler displays `mainGroup` first and switches to `recurrenceGroup` when users change the "Repeat" drop-down. Configure **editing**.[form](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/form/) and **editing**.[popup](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/popup/) to customize the form layout.
```

Comment:
Maybe 
Configure **editing**.[popup](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/popup/) to customize edit form's dialog options and **editing**.[form](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/form/) to customize the form's layout.